src/components/BookmarkList.tsx

```
@@ -0,0 +1,288 @@
+
+          <div className="absolute top-4 right-4 menu-container opacity-0 group-hover:opacity-100 transition-all duration-200">
+            <div
+              onClick={(e) => handleMenuClick(url.id, e)}
+              className="p-2.5 bg-white/90 hover:bg-white border border-slate-200/60 rounded-xl shadow-lg shadow-slate-500/10 hover:shadow-slate-500/20 transition-all duration-200 backdrop-blur-sm cursor-pointer"
+              title="メニュー"
+            >
+              <svg
+                className="w-4 h-4 text-slate-600"
+                fill="none"
+                stroke="currentColor"
+                viewBox="0 0 24 24"
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)